editors/sc-ide/widgets/settings/sclang_page.cpp

```diff
@@ -76,7 +77,6 @@ void SclangPage::load( Manager *s )
 
     ui->autoStart->setChecked( s->value("autoStart").toBool() );
     ui->runtimeDir->setText( s->value("runtimeDir").toString() );
-    ui->sclang_standalone_mode->setChecked(s->value("standalone").toBool());
```

Comment:
Since this setting is now saved in the language file, I reasoned that it didn't make sense to also save it in the IDE preferences. The widget is checked or not based on the value saved in the lang config file.